Multisig/Presentation/UI Library/Texts.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Texts.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 14.04.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+extension Font {
```

Comment:
I would discuss with Sasha using system fonts as much as possible. Is it possible to configure a global style of system fonts, by the way? (I think it should be)
Did you try to run it in dark mode?

Reply:
> system fonts
These fonts are from company-wide-based design style

> dark mode
I turned off dark mode for the app